ports/stm32/system_stm32.c

```diff
@@ -587,6 +587,20 @@ MP_WEAK void SystemClock_Config(void) {
     // Enable the Debug Module in low-power modes.
     DBGMCU->CR |= (DBGMCU_CR_DBG_SLEEPD1 | DBGMCU_CR_DBG_STOPD1 | DBGMCU_CR_DBG_STANDBYD1);
     #endif
+
+    // Configure the analog switches
+    #if MICROPY_HW_ANALOG_SWITCH_PA0
```

Comment:
I think this should be `#ifdef ...` so that the code below runs in all cases that this macro is defined to something.

Reply:
Fixed.